lib/source.re

```diff
@@ -25,8 +25,16 @@ let to_local_path: t => local =
       tempdir;
     };
 
+let is_local_template_dir = s => {
+  Utils.Filename.(
+    test(Exists, s)
+    && test(Exists, join([s, "/spin"]))
+    && test(Exists, join([s, "/template/spin"]))
```

Comment:
`join` is going to put the appropriate separators, so you can write:
```
    && test(Exists, join([s, "spin"]))
    && test(Exists, join([s, "template", "spin"]))
```

Reply:
🤦‍♂ I had not pushed the final version, sorry for that.